TeamCode/src/main/java/org/firstinspires/ftc/teamcode/servoWrist.java

```diff
@@ -0,0 +1,26 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+
+public class servoWrist {
+    public Servo servoWrist;
+
```

Comment:
Yeah, I agree with Soren that a lot of things have been done right. With spacing its kind of hard to tell on GitHub, however we do use tabs rather than spaces when indenting code. Otherwise like Soren said spacing is nice and consistant

Reply:
Agree with Tony, use tabs and be liberal with line breaks for easy readability.

Reply:
Agree with Abbot as well on this one